### PR TITLE
YLP-1945 Migrate NoBar component to dumb

### PR DESCRIPTION
**Pull request by TimCanu**

### CODE REVIEW

File: packages/dumb/src/components/stats/total-insulin/total-insulin-stat.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  title: string
+  total: number
+}
+
+const TotalInsulinStat: FunctionComponent<TotalInsulinStatProps> = (props) => {
+  const { annotations, data, title, total } = props
+
+  const percent = (value: number): string => {
+    const res = Math.round(100 * value / total)
+    return res > 0 ? res.toString(10) : '--'
```
Comment (fatb38) [inline packages/dumb/src/components/stats/total-insulin/total-insulin-stat.tsx:52]: Not mandatory to convert to string ? The number will be convert in the span

Comment (TimCanu) [reply packages/dumb/src/components/stats/total-insulin/total-insulin-stat.tsx:52]: The method return type is string

Comment (fatb38) [reply packages/dumb/src/components/stats/total-insulin/total-insulin-stat.tsx:52]: and what about to type `string | number`  😅

**Review by fatb38 — APPROVED** [review]

**Pull request merged**